module/documents/chat-message.mjs

```diff
@@ -373,10 +373,25 @@ export default class ChatMessage5e extends ChatMessage {
    */
   _enrichAttackTargets(html) {
     const attackRoll = this.rolls[0];
-    const targets = this.getFlag("dnd5e", "targets");
     const visibility = game.settings.get("dnd5e", "attackRollVisibility");
     const isVisible = game.user.isGM || (visibility !== "none");
-    if ( !isVisible || !(attackRoll instanceof dnd5e.dice.D20Roll) || !targets?.length ) return;
+    if ( !isVisible || !(attackRoll instanceof dnd5e.dice.D20Roll) ) return;
+
+    const masteryConfig = CONFIG.DND5E.weaponMasteries[attackRoll.options.mastery];
+    if ( masteryConfig ) {
+      const p = document.createElement("p");
+      p.classList.add("supplement");
+      let mastery = masteryConfig.label;
+      if ( masteryConfig.reference ) mastery = `
+        <a class="content-link" draggable="true" data-link data-uuid="${masteryConfig.reference}"
+           data-tooltip="${mastery}">${mastery}</a>
+      `;
+      p.innerHTML = game.i18n.format("DND5E.WEAPON.Mastery.Flavor", { mastery });
```

Comment:
I don't think this should be done via localisation, we wouldn't expect the word order to change here based on language since it's in `key: value` format. That probably goes for 'on save' as well.

```suggestion
      p.innerHTML = `<strong>${game.i18n.localize("Mastery")}</strong> ${mastery}`;
```